.github/workflows/sphinx.yml

```
@@ -34,7 +34,7 @@ jobs:
       - name: Set Up Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.x
+          python-version: 3.9
```

```suggestion
          python-version: "3.9"
```

When specifyinh version 3.10, not using quotes mean we get version 3.1. lets avoid that ahead of time by providing quotes.

---

Didn't know about this, thank you!